site/content/docs/5.2/components/toasts.md

```diff
@@ -215,7 +215,7 @@ Place toasts with custom CSS as you need them. The top right is often used for n
     </select>
   </div>
 </form>
-<div aria-live="polite" aria-atomic="true" class="bg-dark position-relative bd-example-toasts">
+<div aria-live="polite" aria-atomic="true" class="bg-body-secondary position-relative bd-example-toasts rounded-3">
```

Comment:
IDK if they are voluntary changes but the 3 "Placement" examples were before on a black background. Now the first one is on a `.bg-body-secondary` here and the 2 others don't have this same background color.

Reply:
This is a bit of a special use case—we have a live area inside the example and I wanted to continue to highlight that, but in a responsive to color mode way.